Makefile

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+.PHONY: format
+format:
+	yapf -i --style='{based_on_style: google, indent_width: 2}' -r pipeline_dp tests
```

Comment:
According to Google style guide indent is 4 spaces
https://google.github.io/styleguide/pyguide.html#34-indentation

Reply:
Apologies! I thought it was changed to `2`. I updated the Makefile and ran `yapf` for 4 spaces 👍